cmd/bro-backend/main.go

```diff
@@ -3,14 +3,19 @@ package main
 import (
 	"log"
 
+	brobackend "github.com/threeal/bro/pkg/bro-backend"
```

Comment:
Ooh, for this one, i mean we put `config.go` inside this `cmd/bro-backend` directory.
So it's the same directory as this `main.go`.

And since it's inside `cmd`, we could just skip testing here.
Let's focus testing on `pkg/utils/config.go` instead.

If we had any same pattern between `bro-backend/config.go` and `bro/config.go` let's merge that to `utils/config.go`, and add tests if necessary.

The same applies with `pkg/bro`.